homework.md

```diff
@@ -92,22 +92,22 @@ A place for us to store and share our code.
 
 What does it mean to "Commit your changes"?
 
-Send you code to Github
+Save changes and updates to your code.
```

Comment:
Hi Cheslin I think you should clarify where it saves your changes and updates. but well done